src/main/java/com/echobox/api/linkedin/types/Share.java

```diff
@@ -24,9 +24,9 @@
 import lombok.Setter;
 
 /**
- * Share POJO
+ * ShareStatistics POJO
```

Comment:
Have been over-zealous on the find and replace? This is still the `Share` class, please check all comment changes that have changes `Share` -> `ShareStatistics` as many of these look wrong.